packages/back-end/src/models/ExperimentModel.ts

```diff
@@ -853,5 +881,141 @@ export const getAllVisualExperiments = async (
     visualChangeset: c,
   }));
 
-  return visualExperiments.filter(isValidVisualExperiment);
+  return visualExperiments.filter(_isValidVisualExperiment);
+};
+
+const _getPayloadKeys = (
+  organization: OrganizationInterface,
+  experiment: ExperimentInterface
+): SDKPayloadKey[] => {
+  const environments =
+    organization.settings?.environments?.map((e) => e.id) ?? [];
+  const project = experiment.project ?? "";
+  return environments.map((e) => ({
+    environment: e,
+    project,
+  }));
+};
+
+const _getAllVisualChanges = async (
+  organization: OrganizationInterface,
+  experiment: ExperimentInterface
+) => {
+  const visualChangesets = await findVisualChangesetsByExperiment(
+    experiment.id,
+    organization.id
+  );
+
+  if (!visualChangesets.length) return [];
+
+  return flatten(visualChangesets.map((vc) => vc.visualChanges));
+};
+
+const _hasVisualChanges = async (
+  organization: OrganizationInterface,
+  experiment: ExperimentInterface
+) => {
+  const visualChanges = await _getAllVisualChanges(organization, experiment);
+  return visualChanges.some((vc) => !!vc.css || !!vc.domMutations.length);
+};
+
+const _fireWebooks = async (
```

Comment:
We can get rid of this function entirely for experiments. The webhooks and CDN invalidate have been deprecated for a while now and aren't supported anymore.

Reply:
Cleaned up